helm_drydock/drivers/__init__.py

```diff
@@ -11,11 +11,92 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Yes, found this about half way through when I started seeing import loops. I'll refactor away from __init__.py holding anything other than possible namespace populators.